examples/aws-container-registry/go/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.21
 require (
 	github.com/pulumi/pulumi-aws/sdk/v5 v5.30.0
 	github.com/pulumi/pulumi-docker/sdk/v4 v4.0.0-alpha.4
-	github.com/pulumi/pulumi/sdk/v3 v3.101.1
+	github.com/pulumi/pulumi/sdk/v3 v3.99.0
```

Comment:
Downgrade?

Reply:
I think you were looking at the changes between revisions, this is upgrading from 3.55 to 3.99.

Reply:
LGTM then thank you!